src/extension/features/accounts/last-reconciled-date/index.js

```diff
@@ -4,7 +4,7 @@ import { controllerLookup } from 'toolkit/extension/utils/ember';
 import { getEntityManager } from 'toolkit/extension/utils/ynab';
 const YNAB_ACCOUNTS_HEADER_RIGHT = '.accounts-header-balances-right';
 const TK_LAST_RECONCILED_ID = 'tk-last-reconciled-date';
-
+const TK_DAYS_SINCE_RECONCILED_ID = 'tk-days-since-reconciled';
```

Comment:
nit: can we add a line feed after the const and class declaration

Reply:
done